roles/download/defaults/main.yml

```diff
@@ -163,11 +163,18 @@ crictl_checksums:
 # Kubernetes versions above Kubespray's current target version are untested and should be used with caution.
 kubelet_checksums:
   arm:
+    v1.24.1: 393d130a1715205a253b2f70dbd1f00d1a52ab89b4f3684ed116a937e68116ec
```

Comment:
2.18 wil definitely not support 1.24, please remove references to this version in a back port to release-2.18 branch.

Reply:
Done